server/app/routers/ai.py

```
@@ -0,0 +1,39 @@
+from fastapi import APIRouter, UploadFile, File, Form
+from fastapi.responses import JSONResponse
+import tempfile, os
+from app.core.config import settings
+from app.utils.analysis import SpermAnalyzerService
+
+router = APIRouter(prefix="/ai/sperm", tags=["ai"])
+
+analyzer = SpermAnalyzerService(settings.YOLO_MODEL_PATH)
+
+@router.post("/analyze-image")
+async def analyze_image(file: UploadFile = File(...)):
```

Ruff (0.12.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)